R/draw_heatmap.R

```diff
@@ -0,0 +1,93 @@
+#' Heatmap for Gene Expression Counts
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' This produces a heatmap of the normalized gene expression counts and groups by various factors available in the assay dataset
```

Comment:
```suggestion
#' This produces a heatmap of the chosen assay and groups by various sample variables.
```
because it does not have to be only factors actually - the annotation could also be numeric variables